roofit/roofitcore/src/RooAbsData.cxx

```diff
@@ -112,7 +113,7 @@ Bool_t RooAbsData::releaseVars(RooAbsData* data)
 RooAbsData::RooAbsData()
 {
   claimVars(this) ;
-  _dstore = 0 ;
+  _dstore    = nullptr;
```

Comment:
you mean we don't need explicit initialising at all or replacing `0` by `nullptr`?